temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflowContext.java

```diff
@@ -723,7 +725,14 @@ private StartChildWorkflowExecutionParameters createChildWorkflowParameters(
 
     Map<String, Object> searchAttributes = options.getSearchAttributes();
     if (searchAttributes != null && !searchAttributes.isEmpty()) {
+      if (options.getTypedSearchAttributes() != null) {
+        throw new IllegalArgumentException(
+            "Cannot have typed search attributes and search attributes");
```

Comment:
Just a touch clearer -- applies in a few places
```suggestion
            "Cannot have both typed search attributes and search attributes");
```

Reply:
Updated